src/view/app/Query/query.tsx

```diff
@@ -696,11 +719,22 @@ Description: ${errorObject.description}`}</pre>
                                     }}
                                     onKeyDown={handleKeyDown}
                                 />
-                                <ProBroButton
-                                    ref={(input) => (inputQuery = input)}
-                                    startIcon={<PlayArrowTwoToneIcon />}
-                                    onClick={onQueryClick}
-                                >Query</ProBroButton>
+
+                                {isWindowSmall ? (
+                                    <ProBroButton
+                                        ref={(input) => (inputQuery = input)}
+                                        startIcon={<PlayArrowTwoToneIcon />}
+                                        onClick={onQueryClick}
+                                    />
+                                ) : (
+                                    <ProBroButton
+                                        ref={(input) => (inputQuery = input)}
+                                        startIcon={<PlayArrowTwoToneIcon />}
+                                        onClick={onQueryClick}
+                                    >
+                                        Query
+                                    </ProBroButton>
+                                )}
```

Comment:
maybe it is possible to make query textbox change size by window width so that  query button never moves down?